src/diagonal.works/b6/ingest/mutable.go

```diff
@@ -355,22 +373,6 @@ func (m *BasicMutableWorld) RemoveTag(id b6.FeatureID, key string) error {
 	return nil
 }
 
-func wrapFeature(f Feature, w b6.FeaturesByID) b6.PhysicalFeature {
```

Comment:
Turns out I made a public implementation of this, WrapFeature, and I just forgot to delete this private copy.

Reply:
I c u re saving tests for internal testing round ;)